samples/Mvc.ControllerForMultipleAreasSample/Areas/Services/Views/Home/Index.cshtml

```diff
@@ -0,0 +1 @@
+﻿Services View
```

Comment:
Need a view for `Manage`?

Reply:
https://github.com/aspnet/Entropy/pull/262/files#diff-e9ab401e3324db45ac2bc4aa3f860381